### ISSUE TITLE
Accessibility insights (WCP) - Ability to Identify duplicate errors (Parent & sub-URLs)

### ISSUE THREAD

**Issue by chjaya-a11y** [microsoft/accessibility-insights-service#749]
#**# Is your feature request related to a problem? Please describe.**

Accessibility insights that is integrated in the WCP Portal, performs scans on websites and shows the cumulative error number. The scan is unable to detect duplicate errors that appears across all pages and show them.Due to this, the error numbers looks way too inflated. 

For example, if you to the WCP portal (aka.ms/wcp) and search for powerapps.microsoft.com and go to compliance results, the scan shows a total of 575 errors (includes errors from sub-urls under the parent url)  but the site owner mentioned that there are only 5 errors that the team required to fix to resolve all the 575 errors. They had to manually look for each single error to identify duplicates which is a cumbersome process.

As CELA plans to enable scans for all websites registered across Microsoft, this feature will extremely benefit the teams to efficiently work on specific issues instead of having to spend time identifying duplicates.

## **Describe the solution you'd like**

Add ability in the accessibility insight to identify duplicate errors across all sites registered under the parent URL to show only duplicate errors. 

## **Describe alternatives you've considered**
None. Site owners have to manually click on every single website that are registered under parent URL to understand what is duplicated. 

## Additional context

https://webcompliance.trafficmanager.net/app/ComplianceResults#/d3f050b0-9992-e711-80c4-0004ffc10019/PowerApps Marketing Website

Please reach out to me for additional questions.

**Comment by animania4ka** [comment]
Transitioned to our internal board.
